frontend/ui/data/koptoptions.lua

```diff
@@ -110,6 +110,19 @@ In 'semi-auto' and 'manual' modes, you may need to define areas once on an odd p
                 name_text_hold_callback = optionsutil.showValues,
                 help_text = _([[Set margins to be applied after page-crop and zoom modes are applied.]]),
             },
+            {
+                name = "auto_straighten",
+                name_text = _("Auto Straighten"),
+                toggle = {_("0°"), _("5°"), _("10°"), _("15°"), _("25°")},
+                values = {0, 5, 10, 15, 25},
+                event = "DummyEvent",
+                args = {0, 5, 10, 15, 25},
+                more_options = true,
```

Comment:
Is a "more options" fine tuning button really needed ? Would I want to force it to be 12° as I wouldn't be fine with 10° or 15° ?